MangoAPI.BusinessLogic/ApiCommands/Users/ChangePasswordCommandValidator.cs

```diff
@@ -29,6 +29,7 @@ private static bool PasswordIsStrong(string pass)
                && pass.Any(char.IsUpper)
                && pass.Any(char.IsLower)
                && pass.Any(char.IsDigit)
-               && pass.Any(char.IsSymbol);
+               && pass.Any(char.IsSymbol)
```

Comment:
Please move this method to another class, like `PasswordValidator`. Also create unit tests for `PasswordValidator` class using `Theory` and `InlineData` attributes of `XUnit` 

See example at https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata